packages/compartment-mapper/src/bundle.js

```diff
@@ -245,7 +245,13 @@ export const makeBundle = async (read, moduleLocation, options) => {
   for (const module of modules) {
     module.indexedImports = Object.fromEntries(
       Object.entries(module.resolvedImports).map(([importSpecifier, key]) => {
-        key = aliases.get(key) ?? key;
+        // TODO Change to the following line, once our tools don't choke on `?.`.
```

Comment:
consider an `UNTIL <issue>` comment https://github.com/Agoric/agoric-sdk/wiki/Coding-Style#code-comments

The issue provides enough documentation